3-0-spring-framework/3-3-0-enable-string-trimming/src/test/java/com/bobocode/TrimmingNotEnabledTest.java

```diff
@@ -0,0 +1,36 @@
+package com.bobocode;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.bobocode.config.TrimmingNotEnabledConfig;
+import com.bobocode.service.TrimmedService;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.junit.jupiter.SpringJUnitConfig;
+
+@SpringJUnitConfig(classes = TrimmingNotEnabledConfig.class)
+class TrimmingNotEnabledTest {
```

Comment:
@Stanislav-Zabramnyi Let's move them all in one bit test suite (user nested test classes). 
1. Configuration
2. Bean Post Processor